wiki/Contributing.md

```diff
@@ -0,0 +1,86 @@
+# Contributing
+
+Thank you for considering contributing to our project! We appreciate your interest and support. To make the process as smooth as possible, please follow the guidelines below.
+
+## Requirements
+
+- You must have [Godot version 4.2.2](https://godotengine.org/download/archive/4.2.2-stable/) installed as this project is made in godot (Windows, Linux, Mac)
+- Git installed on you computer. Install git by [clicking here](https://git-scm.com/) (Windows, Linux, Mac)
+
+if you are a linux user you can also install Git through your package manager
+- Debian-based: 'sudo apt install git'
+- RedHat-based: 'sudo dnf install git'
+- Arch-based: 'sudo pacman -S git'
+- openSUSE: 'sudo zypper install git'
+- Alpine: 'sudo apk add git'
+- FreeBSD: 'sudo pkg install git'
+
+## How to contribute
+
+### Getting Started
+
+1. Fork the repository: Click the 'Fork' button at the top right corner of the repository page.
+
+2. Clone your forked repository:
+
+> git clone https://github.com/quellus/GDTuber.git
+
+3. Navigate to the project directory:
+
+> cd REPO_NAME
+
+4. Create a new branch for your feature or bugfix:
+
+> git checkout -b feature-or-bugfix-name
+
+
+### Making Changes
+
+1. Open the project in Godot (version 4.2.2).
+   
+2. Make your changes or additions in the Godot editor.
+   
+3. Test your changes thoroughly to ensure they work as expected.
+
+### Committing Your Changes
+
+1. Add your changes:
+
+> git add .
+
+2. Commit your changes with a descriptive commit message:
+
+> git commit -m "Description of changes"
+
+### Pushing Your Changes
+
+1. Push to your forked repository:
+
+> git push origin feature-or-bugfix-name
+
+2. Create a Pull Request:
+
+1. Go to the [GDTuber](https://github.com/quellus/GDTuber/) repository.
+2. Click on the 'Pull requests' tab.
+3. Click the 'New pull request' button.
+4. Select the branch you pushed your changes to in your forked repository.
+5. Click 'Create pull request' and fill out the required information.
```

Comment:
The numbering gets a little weird with this one. I think we need to tab these over to make it work right

```suggestion
  1. Go to the [GDTuber](https://github.com/quellus/GDTuber/) repository.
  2. Click on the 'Pull requests' tab.
  3. Click the 'New pull request' button.
  4. Select the branch you pushed your changes to in your forked repository.
  5. Click 'Create pull request' and fill out the required information.
```